DemoQA/Pages/webTablesPage.js

```diff
@@ -58,9 +65,51 @@ class WebTablesPage {
 
     async enterSearchText(string) {
         await t
-          .typeText(this.searchBar, string);
+            .typeText(this.searchBar, string);
+    }
+
+    async openPageSize() {
+        await t.click(this.pageSizeOption);
+    }
+
+    async selectPageSize(pageSize) {
```

Comment:
Actually this function is duplicated and should be deleted, right? You have the selectPageSize function which makes this one obsolete.